src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -13,26 +13,51 @@ Starting April 12, 2021, we're allowing some customers who have users on our ori
 * Users can be assigned to one or more groups (for example, our default **Admin** group or a custom group like **Contractors**). For large organizations, users are often in multiple groups. 
 * When you want to add new groups, or give groups access to more accounts, you must create an access grant. This assigns a group access to: 1) a specific role and 2) a specific account. For example, you may give a Contractors group access to our default **All product admin** role on one or more of your accounts, or give that group a custom role (a role being a custom set of [capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts)). 
 
-The more complex your organization is, the more it may help you to first plan out how your user groups and access grants will be structured. 
+To read more about these concepts, see [User management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts).
+
+
+## Think about what groups you'll need [#plan-groups]
+
+Once you have a decent understanding of the user management concepts, you'll want to think about how you'll implement groups for your organization. The more complex your organization is, the more it may help you to first plan out how your user groups and access grants will be structured. 
+
+Here are some tips for how different types of organizations may use groups:  
+
+* For organizations with multiple accounts, a common configuration would include a group with the **Organization manager** role, a group with the **Authentication domain manager**, a group with the **Billing user** at the master account level, and then configuring product admins, users, and all other roles at the sub-account level, depending on organization preferences.  
+* Customers with smaller or flatter organizations that require transparency for all teams across all of their data could have as little as 1 or 2 groups.
+* For organizations where all data is in a single account, a common configuration might be around five or six groups.
+
+As an optional step, we've created [a planning spreadsheet](https://docs.google.com/spreadsheets/d/1-0saEkEFXnLjwjLXhoSAEAP9lbU0iaZzjMEraGNJOME/edit?usp=sharing) to help you plan out the groups and access grants you'll need. This is only for planning purposes as a reference guide: it's not intended to be uploaded. Please make a copy and follow the instructions. At the appropriate point in the user migration wizard workflow, you can use this planning resource to help you create user groups and access grants. 
```

Comment:
It might be helpful to describe a little bit more about what the migration wizard workflow looks like.

Reply:
Or at the very least you could add a note saying that you can only use the migration wizard if you have the correct account permissions.